### ISSUE TITLE
Can't authorize extension in Linux

### ISSUE THREAD

**Issue by doughsay** [Microsoft/vscode-pull-request-github#579]
- Extension version: **0.2.1**
- VSCode Version:

```
Version: 1.28.1
Commit: 3368db6750222d319c851f6d90eb619d886e08f5
Date: 2018-10-11T18:09:20.566Z
Electron: 2.0.9
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64
```

- OS: **Linux / Manjaro**

Steps to Reproduce:

1. Click "Sign in" in extension pop-up
2. Browser opens
4. Click "Continue" (because I'm already logged in to GitHub in my browser)
5. Review permission and grant them (only happens first time)
6. Nothing happens / extension is not authorized

I've tried using both Chrome (69.0.3497.100) and Firefox (62.0.3).  The first time I tried in Chrome it asked to invoke xdg-open and I checked the box to always do this.

Is there a way I can manually authorize the extension?  Right now I can't use it at all in Linux.

**Comment by doughsay** [comment]
@queerviolet thanks for the instructions!  That's easy enough.  I did not install via snap, I installed it via Manjaro's package manager `pacman` from the AUR: https://aur.archlinux.org/packages/visual-studio-code-bin

**Comment by cbrnr** [comment]
Same here (Arch Linux), I think the problem is that once you grant permission in the browser, the command used to invoke Code does not work. Linux is using xdg-open to associate file types with applications, and apparently nothing is defined in this case.

**Comment by queerviolet** [comment]
Hi! I think this flow should be fixed by https://github.com/Microsoft/vscode-pull-request-github/pull/659

The uri handler may not open, but the auth server and newest version of the extension now provide an alternative route.

Please open a new issue if there are any problems.

**Comment by tunnckoCore** [comment]
Great! I'll try it again.

**Comment by cbrnr** [comment]
I just tried to authorize, but nothing really changed from a user perspective. Nothing happens if I click Continue and then Open with xdg-open.

I noticed that I can click on the "Signing in to github.com" entry in the status bar, upon which I am prompted to enter a token. However, it is just not super clear for the user how to do this - maybe a link to the documentation could be included in the message (right now, the message just says: "Token (Press 'Enter' to confirm or 'Escape' to cancel)").
